README.md

```diff
@@ -1,9 +1,7 @@
-# Bubberstation config overrides
-On the live server these files replace any files of the same name within the config/ subfolder of the repo. 
-This is necessary because testing on localhost necessarily requires different configs than the live servers.
-To facilitate this this repo is pulled into GameStaticFiles/config/ by TGS at the end of every round(via a helper EventScript).
-Actually this is not the case yet, due to difficulties in doing so...
+# Bubberstation Job Limits Increase Fork
+This is a fork to increase job limits as indicated by the name.
```

Comment:
Any changes made yon your branch will be in the PR- to avoid this you're supposed to use branches but here I'd just edit it back. The idea is that you make a branch(which is functionally a separate version of the files) for each PR you make and then PR from that branch.

Reply:
Apologies, literally first PR. I'll fix that by reverting the file